modules/ROOT/pages/index.adoc

```diff
@@ -37,9 +37,63 @@ Now consider that this organization also wants to provide a web application to a
 By using Anypoint DataGraph, the organization does not need to create a new API for every new business requirement. The enterprise architect can discover the existing order, shipment, and product APIs and add each API schema to Anypoint DataGraph. Developers can then request access to the unified schema and query the data that they need or update it from a single endpoint. 
 
 == Anypoint DataGraph API Support
-Anypoint DataGraph supports:
+
+DataGraph supports the following API types:
 
 * REST APIs with RAML and OAS specifications
+* GraphQL APIs
+* Apollo GraphQL APIs
+
+=== GraphQL Feature Support
```

Comment:
@gbattist I added those two tables here. We can remove the items that aren't supported